data/movements/scripts/tiles.lua

```diff
@@ -25,7 +25,7 @@ function onStepIn(creature, item, position, fromPosition)
 	if Tile(position):hasFlag(TILESTATE_PROTECTIONZONE) then
 		local lookPosition = player:getPosition()
 		lookPosition:getNextPosition(player:getDirection())
-		local depotItem = Tile(lookPosition):getItemByType(ITEM_TYPE_DEPOT)
+		local depotItem = lookPosition:getTile():getItemByType(ITEM_TYPE_DEPOT)
```

Comment:
Revert this, no need to go through another function when you can just use Tile(position).